### DISCUSSION TITLE
Allow `import com.example.blah.* with given`

### DISCUSSION THREAD

**Discussion by japgolly** [scala/scala3#17866] [Feature Requests]
Instead of this:

```scala
import com.example.blah.{*,given}
```

how about we support

```scala
import com.example.blah.* with given
```

?

**Comment by bishabosha** [comment]
does this only apply for `given Foo`, or replace all commas?

**Comment by som-snytt** [comment]
Also another opportunity to work in `import p.* for given`.

Arguably, implicits are Scala's original sin.
